x-pack/plugins/observability_solution/entity_manager/server/lib/entities/install_entity_definition.ts

```diff
@@ -218,3 +188,123 @@ async function installAndStartDefinition(
   }
   return definition;
 }
+
+// perform installation of an entity definition including all
+// the necessary components
+async function install({
+  esClient,
+  soClient,
+  definition,
+  logger,
+}: InstallDefinitionParams): Promise<EntityDefinition> {
+  logger.debug(
+    () =>
+      `Installing definition ${definition.id} v${definition.version}\n${JSON.stringify(
+        definition,
+        null,
+        2
+      )}`
+  );
+
+  const entityDefinition = await saveEntityDefinition(soClient, {
+    ...definition,
+    installStatus: 'installing',
+    installStartedAt: new Date().toISOString(),
+  });
+
+  logger.debug(`Installing index templates for definition ${definition.id}`);
+  await Promise.allSettled([
+    upsertTemplate({
+      esClient,
+      logger,
+      template: getEntitiesHistoryIndexTemplateConfig(definition),
```

Comment:
Totally out of scope for this PR:
For the other EEM components we use the word `generate` rather than `get` 🤷🏼

Reply:
I'll update

Reply:
fixed in https://github.com/elastic/kibana/pull/188351/commits/3e97368de00abd8b726d884657b2184a34870e28